mne/fixes.py

```diff
@@ -1091,3 +1091,61 @@ def mean(array, axis):
     @contextmanager
     def nullcontext(enter_result=None):
         yield enter_result
+
+
+###############################################################################
+# Work around autosummary making bad :class: links
```

Comment:
this sucks. It was working with an older version? if so can we rather pin the version rather than crippling our code?

Reply:
How does this cripple our code? It just replaces a few docstrings. Subclassing like `isinstance(x, dict)` are still fine. Overhead should be minimal.

The problem is that numpydoc improved the signature extraction, and we hit a bug now with autosummary trying to add effectively:
```
:class:`v, remove specified key and return the corresponding value.`
```
to correspond to the `D.pop() -> v, remove specified key and return the corresponding value.` docstring.

This seemed like the cleanest solution, but I'm open to other ideas.